packages/react-storage/src/components/StorageManager/ui/FileList/__tests__/FileControl.test.tsx

```diff
@@ -101,4 +101,19 @@ describe('FileControl', () => {
 
     expect(container).toMatchSnapshot();
   });
+
+  it('should default to showThumbnails being true', () => {
+    //@ts-expect-error
```

Comment:
Huh, looks like something prettier should have caught

Reply:
The spacing?

Reply:
The `//@ts-expect-error`? Scott suggested I use it here.

`showThumbnails` is required, but is assigned a default value anyways. This test adds coverage for that default value assignment, since it would be inaccessible without a type error due to a missing required prop.